src/main.cpp

```diff
@@ -1324,12 +1324,27 @@ bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex)
 
 CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
 {
```

Comment:
This function is used both for the miner to compute the subsidy and for the consensus rule check, right? Just making sure we're not forgetting to change it elsewhere.